src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -495,7 +494,7 @@ public ActivityProfileState isActive( final AlignmentContext context, final Refe
             // TODO: now that old qual is gone, do we still need this?
```

Comment:
Is this todo still relevant?

Reply:
PR #6343 addressed it, so the TODO will be gone after rebasing.